datadog_lambda/wrapper.py

```diff
@@ -105,24 +118,59 @@ def __call__(self, event, context, **kwargs):
 
     def _before(self, event, context):
         try:
+
             set_cold_start()
             submit_invocations_metric(context)
             # Extract Datadog trace context from incoming requests
-            extract_dd_trace_context(event)
+            dd_context = extract_dd_trace_context(event)
+
+            self.span = None
+            if dd_tracing_enabled:
+                self.span = self._create_dd_trace_py_span(context, dd_context)
```

Comment:
Nits, i feel this function is more about "creating a span for the function execution" rather than "creating a dd trace py span".  Perhaps `create_function_execution_span` or something better?